app/controllers/claimToAdjustPoa/AmendablePOAController.scala

```diff
@@ -52,12 +56,13 @@ class AmendablePOAController @Inject()(val authorisedFunctions: AuthorisedFuncti
         ifAdjustPoaIsEnabled(isAgent) {
           claimToAdjustService.getPoaForNonCrystallisedTaxYear(Nino(user.nino)) flatMap {
             case Right(Some(paymentOnAccount: PaymentOnAccountViewModel)) =>
-              Future.successful(
+              isSubsequentAdjustmentAttempt(paymentOnAccount) map { poasHaveBeenAdjustedPreviously =>
                 Ok(view(
                   isAgent = isAgent,
-                  paymentOnAccount = paymentOnAccount
+                  paymentOnAccount = paymentOnAccount,
+                  poasHaveBeenAdjustedPreviously = poasHaveBeenAdjustedPreviously
```

Comment:
looks like you need to create a separate view model to incorporate poasHaveBeenAdjustedPreviously field;
you can create a new method in ClaimToAdjustService which execute getPoaForNonCrystallisedTaxYear and isSubsequentAdjustmentAttempt logic to return a new view model;